modules/persistent-storage-local-removing-devices.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-local.adoc
+
+[id="local-removing-device_{context}"]
+= Removing a local volume
+
+It is occasionally needed to delete a local volume. While removing the entry in the LocalVolume resource and deleting the PersistentVolume is typically enough, if you want to re-use the same device path or have it managed by a different StorageClass, then additional steps are needed.
```

Comment:
The first sentence seems awkward and might be difficult to translate. Maybe something like "You occasionally need to delete a local volume" or "Occasionally, local volumes must be deleted."